index.html

```diff
@@ -1,29 +1,96 @@
+            Psychologiczno-Pedagogicznym
+            odpowiedzialnym za wydawanie orzeczeń.</p>
+        <p class="decription__text">Idea powstała, dzięki Pani Alince wykonującą tę pracę. Wiąże się ona ze żmudnym
+            wypełnianiem tych samym
+            danych w wielu dokumentach, a tym samym prowadzi do godzin niepotrzebnej pracy. Chcąc wesprzeć cały
+            proces stworzyliśmy narzędzie, które zautomatyzuje generowanie dokumentów zgodnych z oficjalnymi
+            standardami Ministerstwa Edukacji Narodowej. Tym samym wzrośnie efektywność oraz zmaleje ilość błędów w
+            porównaniu z ręcznym wprowadzaniem danych.</p>
+        <p class="decription__text">Generowany dokument zostanie utworzony jako plik .docs, możliwy do edycji w Wordzie
+            i ewentualnego
+            dopisania dodatkowych informacji.</p>
+        <img src="assets/images/description-graphics.svg" alt="" class="description__graphics">
+    </section>
+    <section class="demo">
+        <h1 class="titles demo__title">Zobacz Alinkę w akcji</h1>
+        <div class="demo__box">demo</div>
+        <img src="assets/images/wave-left.svg" alt="" class="demo__wave--left">
+        <img src="assets/images/wave-right.svg" alt="" class="demo__wave--right">
+    </section>
+    <section class="features">
+        <dev class="feature features__feature">
+            <img src="assets/images/feature-free.svg" alt="" class="feature__icon">
+            <p class="feature__text">Za darmo</p>
+        </dev>
+        <dev class="feature features__feature">
+            <img src="assets/images/feature-easy.svg" alt="" class="feature__icon">
+            <p class="feature__text">Łatwa w osłudze</p>
+        </dev>
+        <dev class="feature features__feature">
+            <img src="assets/images/feature-safe.svg" alt="" class="feature__icon">
+            <p class="feature__text">Bezpieczna</p>
+        </dev>
+    </section>
+    <section class="download">
+        <h1 class="titles download__title">Zacznij pracować z Alinką!</h1>
+        <div class="download__buttons">
+            <button class="button download__button--color">Pobierz</button>
+            <button class="button download__button--regular">Zmień wersję</button>
+        </div>
+        <img src="assets/images/download-graphics.svg" alt="" class="download__graphics">
+    </section>
+    <footer class="footer">
+        <div class="icons footer__icons">
+            <img src="assets/images/Code_for_Poznan_final-01.png" alt="" class="icons__logo">
+            <p class="icons__email">hello@codeforpoznan.pl</p>
+            <a href="https://www.facebook.com/CodeForPL/" class="icons__facebook fab fa-facebook-f"></a>
+            <a href="https://pl.linkedin.com/company/codeforpoznan" class="icons__linkedin fab fa-linkedin-in"></a>
+            <a href="https://github.com/CodeForPoznan" class="icons__github fab fa-github"></a>
```

Comment:
1. you can group them together as a social media so it will be better to put them into div and also easier to position it by css later on ;)
2. better to use svg instead some external source